benchmarks/run-benchmark.py

```diff
@@ -19,7 +19,7 @@
 import argparse
 from scripts.benchmarks import *
 
-delta_version = "2.1.0"
+delta_version = "2.3.0"
```

Comment:
Aren't we at 2.4 by now? Or is it that you do not want the latest version?

Reply:
EMR didn't upgrade to Spark 3.4 yet so we can't run with Delta 2.4.